### ISSUE TITLE
Add CommonMark (markdown) support to description field in Server Object

### ISSUE THREAD

**Issue by neluzhin** [Rebilly/ReDoc#724]
YAML:

````yaml
url: 'https://api.dev.{base_host}/api/v1'
description: '**Development server** - используйте при активной разработке. Данные стираются после каждого обновления. Присутствуют тестовые данные. Некоторые сервисы работают в "холостом" режиме: например, СМС и электронные письма отправляются не реальным получателям, а в Slack.'
variables:
  base_host:
    default: '__BASE_HOST__'
    description: 'Базовый хост приложения.'
````

What's rendered:

![image](https://user-images.githubusercontent.com/12474739/48955100-5c0aaa80-ef5d-11e8-9c8a-4b57b00dd97f.png)

However [specification says](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/3.0.0.md#server-object) in may support CommonMark:

![image](https://user-images.githubusercontent.com/12474739/48955268-139fbc80-ef5e-11e8-85f0-d729c4c9b6a9.png)

**Comment by viralanomaly** [comment]
I have mine using the pipe syntax like I tried to revise yours to below.  See if that makes a difference?

```
url: 'https://api.dev.{base_host}/api/v1'
description: |
    **Development server** - используйте при активной разработке. Данные стираются после каждого обновления. Присутствуют тестовые данные. Некоторые сервисы работают в "холостом" режиме: например, СМС и электронные письма отправляются не реальным получателям, а в Slack.
variables:
  base_host:
    default: '__BASE_HOST__'
    description: 'Базовый хост приложения.'
```

**Comment by neluzhin** [comment]
I see it was fixed in v2.0.0-rc.0 release. I've checked, ot works well. Thank you!
